source/api/cypress-api/env.md

```diff
@@ -144,7 +172,7 @@ The {% url 'Environment Variables' environment-variables %} guide explains the o
 
 As a rule of thumb anything you call from `Cypress` affects global state. Anything you call from `cy` affects local state.
 
-Since the environment variables added or changed by `Cypress.env` are only in scope for the current spec file, you'd think that it should be `cy.env` and not `Cypress.env`&hellip;and you'd be right. The fact that `Cypress.env` affects local state is an artifact of the API evolving over time: `Cypress.env` used to affect global state&mdash;environment variables added in one test spec file were available in other specs&mdash;but the Cypress team wisely made each spec run in isolation in {% url `3.0.0` changelog#3-0-0 %} and by that time `Cypress.env` was public API.
+Since the environment variables added or changed by `Cypress.env` are only in scope for the current spec file, you'd think that it should be `cy.env` and not `Cypress.env`&hellip; and you'd be right. The fact that `Cypress.env` affects local state is an artifact of the API evolving over time: `Cypress.env` used to affect global state&mdash;environment variables added in one test spec file were available in other specs&mdash;but the Cypress team wisely made each spec run in isolation in {% url `3.0.0` changelog#3-0-0 %} and by that time `Cypress.env` was public API.
```

Comment:
`Minor` `Style`

Adding a space after the ellipses so that the text doesn't look so cramped.